dora/stress/shell/src/main/java/alluxio/stress/cli/worker/StressWorkerBench.java

```diff
@@ -455,35 +456,71 @@ private void runInternal() throws Exception {
       CommonUtils.sleepMs(waitMs);
       SAMPLING_LOG.info("Test started and recording will be started after the warm up at {}",
           CommonUtils.convertMsToDate(recordMs, dateFormat));
+
+      String workerID = mBaseParameters.mIndex;
+      int lastDashIndex = workerID.lastIndexOf("-");
+      if (lastDashIndex != -1) {
+        workerID = toString().substring(lastDashIndex + 1);
+      }
+      WorkerBenchCoarseDataPoint dp = new WorkerBenchCoarseDataPoint(
+          Long.parseLong(workerID),
+          Thread.currentThread().getId(),
+          new ArrayList<>(), new ArrayList<>());
```

Comment:
```suggestion
          new ArrayList<>(), new ArrayList<>());
```
are you going to use these lists and add to them? It's better if you just init empty lists internally.
```
public WorkerBenchCoarseDataPoint(long, long) {
  xxx = new ArrayList<>();
  xxx = new ArrayList<>();
}
```
Kinda weird if you create a list, and give it to someone, you and that someone both hold the list and can add to it.

Reply:
Yes I agree. There are two places creating a `WorkerBenchCoarseDataPoint`. One in the bench (without data and throughput arrays), and another in the deserializer (with data and throughput arrays). I will write two different constructors for them.